src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSubmissionNotAllowed = textField.dataset.isSessionOpen === 'false';
```

Comment:
I think it would be better if you used the jQuery approach for retrieving data. It would be something like this: 
```javascript
$(textField).data('is-session-open')
```

And then, if you pass down `is-session-closed` instead, as I suggested in my comment below, you would not even have to check for equality with a boolean string as jQuery will automatically convert to boolean for you.

Note: I would also recommend changing the variable name to `isSessionOpen` or `isSessionClosed` (depending on the approach we go for) for consistency as `allowed/notAllowed` seems more like a user-role permissions related variable.